optaplanner-persistence/optaplanner-persistence-jpa/src/main/java/org/optaplanner/persistence/jpa/api/score/buildin/simpledouble/SimpleDoubleScoreConverter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jpa.api.score.buildin.simpledouble;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import org.optaplanner.core.api.score.buildin.simpledouble.SimpleDoubleScore;
+
+@Converter
+public class SimpleDoubleScoreConverter implements AttributeConverter<SimpleDoubleScore, String> {
```

Comment:
Let's not add anything for Double scores, as they're getting deprecated (for removal in 8.0).